bitcoin-core-dev-tech/2019-06-06-great-consensus-cleanup.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: Great Consensus Cleanup (2019-06-06)
+title: Great Consensus Cleanup 
 transcript_by: Bryan Bishop
 categories: ['core-dev-tech']
 tags: ['consensus', 'soft fork', 'forks']
+date: 2019-06-06
```

Comment:
speaker Matt Corallo, also mentioned on the linked tweet by kanzure.

Reply:
ping @tvpeter